packages/proto-rpc/src/ServerRegistry.ts

```diff
@@ -0,0 +1,134 @@
+import { RpcMessage } from './proto/ProtoRpc'
+import EventEmitter = require('events')
+import { MethodInfo, RpcMetadata, RpcStatus, ServerCallContext } from '@protobuf-ts/runtime-rpc'
+import { BinaryReadOptions, BinaryWriteOptions } from '@protobuf-ts/runtime'
+import { promiseTimeout } from './common'
+import { Err } from './errors'
+import UnknownRpcMethod = Err.UnknownRpcMethod
+import { Logger } from './Logger'
+import { ConversionWrappers } from './ConversionWrappers'
+import { ProtoRpcOptions } from './ClientTransport'
+import { Empty } from './proto/google/protobuf/empty'
+
+export enum ServerRegistryEvent {
+    RPC_RESPONSE = 'streamr:dht-transport:server:response-new',
+    RPC_REQUEST = 'streamr:dht-transport:server:request-new',
```

Comment:
I don't think there's really a need to have such long values for the events? Maybe just, e.g.:
```suggestion
    RPC_RESPONSE = 'rpcResponse',
    RPC_REQUEST = 'rpcRequest',
```